src/config/config.js

```diff
@@ -20,6 +22,8 @@ const Config = {
     !process.env.REACT_APP_THROW_SAVE_PROFILE_ERRORS || process.env.REACT_APP_THROW_SAVE_PROFILE_ERRORS === 'true',
   withMockedFeeds: process.env.REACT_APP_WITH_MOCKED_FEEDS === 'true',
   safariMobileKeyboardGuidedSize: process.env.REACT_APP_SAFARI_MOBILE_KEYBOARD_GUIDED_SIZE === 'true',
+  receiveUrl: process.env.REACT_APP_RECEIVE_URL || `${publicUrl}/AppNavigation/Dashboard/Send`,
```

Comment:
the default can be $publicUrl for both send and receive since we no longer need the full path right?

Reply:
[done](https://github.com/GoodDollar/GoodDAPP/pull/615/commits/df8a4ab183a99efc9832f9eeba4ad3c9830006ec)